inductiva/__init__.py

```diff
@@ -66,3 +66,31 @@ def _check_for_available_package_update():
 
 
 _check_for_available_package_update()
+
+
+def _supports_ansi():
+    """Checks if we support ansi formatting for colors and bolds"""
+    user_disable_ansi = utils.format_utils.getenv_bool("INDUCTIVA_DISABLE_ANSI",
+                                                       False)
+    if sys.platform.startswith("win"):
+        return "TERM" in os.environ and os.environ[
+            "TERM"] == "xterm" and not user_disable_ansi
+    return hasattr(sys.stdout,
+                   "isatty") and sys.stdout.isatty() and not user_disable_ansi
+
+
+_ansi_enabled = _supports_ansi
+
+
+def get_ansi_formatter():
+    """Fetches the formatter used for ANSI formatting.
+
+    Either `no_formatter` when ansi formatting is disable or the
+    `inductiva.utils.format_utils.emphasis_formatter`
+    
+    """
+    import inductiva
+    if not _ansi_enabled:
+        return inductiva.utils.format_utils.no_formatter
```

Comment:
Can't we use the relative import of the utils here and avoid the disable.

Reply:
Not exactly sure what the best course of action would be here. Maybe @sergiomsantos could shed some light into this.

Reply:
It just seems weird to import exactly the package in the main `__init__.py`

Reply:
let's talk about it.